airflow/example_dags/tutorial.py

```diff
@@ -46,7 +46,7 @@
     # 'on_failure_callback': some_function,
     # 'on_success_callback': some_other_function,
     # 'on_retry_callback': another_function,
-    # 'trigger_rule': u'all_success'
+    # 'trigger_rule': 'all_success'
```

Comment:
it's not a typo but it is useless indeed since the file has the magic line " -*- coding: utf-8 -*-"

Reply:
If we're making this change, can we add `from __future__ import unicode_literals` to the top of this file?

`-- coding: utf-8 --` means that we can use unicode characters in the source code (for example in unicode literals) but it doesn't mean that all strings are treated as unicode. In Python 2.x, this would still be treated as ASCII if we remove the `u''`.

Reply:
Indeed, it is a good practice to add them both for python 2. But I don’t know what the core developers require

Reply:
Airflow uses `from __future__ import unicode_literals` all over the place, so adding it at the top of the file seems consistent with the rest of the code.

Reply:
I would add all these futures directly in all python files:
```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```
isort can do it automatically

Reply:
That seems like a good idea.

Reply:
```
isort -a "from __future__ import print_function" *.py
```